<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrivialPiper</class>
 <widget class="QMainWindow" name="TrivialPiper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cliente - TrivialPiper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="inputIP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaIP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP/HOST</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioU">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload File</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioD">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Download File</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetUpload" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>341</width>
      <height>371</height>
     </rect>
    </property>
    <widget class="QTreeView" name="arbolDir">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>341</width>
       <height>331</height>
      </rect>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>46</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="finalizarU">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QLabel" name="estadoUp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>340</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetDownload" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputDownloadFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="finalizarD">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="estadoDw">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBoxPuerto">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputBlocksize">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="text">
     <string>512</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxBlocksize">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Block Size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPort">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="text">
     <string>69</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxWindowsize">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputWindow">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
